src/app/resources/resources.component.html

```diff
@@ -75,16 +75,17 @@ <h3 class="header"><a *ngIf="!nationName" [routerLink]="['/resources/view', elem
           <span class="stars">
             <mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon>
             <!-- used simple arithmetic to get 20 and converted into % -->
-            <span [ngStyle]="{'width': getRating(element.sum, element.timesRated)}">
+            <span [ngStyle]="{'width': getRating(element.has_rated, 1)}">
               <mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon><mat-icon>star</mat-icon>
             </span>
           </span>
-          <div class="stats stats-total">{{ element.timesRated }} Total</div>
-          <div class="stats stats-female">{{ fRating }}% Female</div>
-          <div class="stats stats-male">{{ mRating }}% Male</div>
+          <div class="stats stats-total">{{ element.total_rating }} Total</div>      
+          <div class="stats"><mat-icon>person-outline</mat-icon>&nbsp;&nbsp;<mat-icon  style="float:right;">person</mat-icon>
```

Comment:
style should go in style.css, double space btn mat-icon and style

Reply:
Fixed,
Thanks @razu9861 
Please review.